fibsem/milling/base.py

```diff
@@ -72,7 +75,7 @@ class FibsemMillingStage:
     pattern: BasePattern = field(default_factory=lambda: get_pattern("Rectangle",
```

Comment:
We can probably also update the default factory lambdas for pattern and strategy to just use DEFAULT_PATTERN and DEFAULT_STRATEGY directly, now that they have default values?

Reply:
That works for DEFAULT_PATTERN but DEFAULT_STRATEGY causes circular imports. That could be avoided by splitting up base.py but that's a bigger change. That said, I was able to use `get_strategy` directly without an argument, avoiding the need for a lambda, since "Standard" is the default.